subdaos/overview.md

```diff
@@ -14,12 +14,15 @@ In most respects, subDAOs are self-standing, unique DAOs. Their semi-independent
 
 ## SubDAO Types
 
-SubDAOs are divided into two types:
+SubDAOs are divided into three types:
```

Comment:
Just wondering where the ProtectorDAOs fit into this new classification? The Allocators explicitly say across "DeFi" - is this meant to include "ReFi" too?